onnxruntime/core/framework/execution_frame.cc

```diff
@@ -292,8 +292,7 @@ ExecutionFrame::ExecutionFrame(const std::vector<int>& feed_mlvalue_idxs, const
             }
 
             // log size of activation. Keep it commented out for now to avoid log flooding.
-            // VLOGS(session_state_.Logger(), 1) << "Allocated memory for activations, size: "
-            //                                   << mem_patterns_->patterns[i].PeakSize();
+             printf("\n **** Allocated memory for activations, size: %zu ***\n", mem_patterns_->patterns[i].PeakSize());
```

Comment:
Can we use VLOGS or similar here?

Reply:
I'll add this in a subsequent PR right after this.

Reply:
https://github.com/microsoft/onnxruntime/pull/5659